docs/ansible/roles/debops.roundcube/getting-started.rst

```diff
@@ -41,10 +41,10 @@ Example playbook
 The following playbook can be used with DebOps. If you are using these role
 without DebOps you might need to adapt them to make them work in your setup.
 
-.. literalinclude:: ../../../../ansible/playbooks/service/roundcube.yml
+.. literalinclude:: ../../../../ansible/playbooks/app/roundcube.yml
```

Comment:
Argh, no... All playbook files are in the `service/` subdirectory, and then they can be symlinked to `app/`, `srv/` etc. And you can always call them like, `debops service/roundcube` this way.